### PR TITLE
Update READMEs and delete deprecated docs

### PR DESCRIPTION
**Pull request by adeatcu-ionos**
Solves https://github.com/ionos-cloud/sdk-nodejs/issues/5 by fixing the example from the `README.md`.

This also represents a follow up for [this](https://github.com/ionos-cloud/sdk-nodejs/issues/6#issuecomment-1259456857) discussion.

In this PR, the READMEs are updated accordingly and the deprecated documentation is deleted.

### CODE REVIEW

**Comment by DanielRuf** [comment]
Hi, there is also one more at https://github.com/ionos-cloud/sdk-nodejs/tree/master/docs#usage

**Comment by DanielRuf** [comment]
Besides that, there is a very old DOCS.md file at https://github.com/ionos-cloud/sdk-nodejs/blob/master/DOCS.md which contains more suche cases.

Not sure if this file is still used or obsolete like the old lowercase files (see https://github.com/ionos-cloud/sdk-nodejs/issues/5#issuecomment-1259487707).

**Comment by adeatcu-ionos** [comment]
> Hi, there is also one more at https://github.com/ionos-cloud/sdk-nodejs/tree/master/docs#usage

Hi, I fixed this typo in [this](https://github.com/ionos-cloud/sdk-nodejs/pull/7/commits/b2d79836818e964c58bcbf48613138eb476e51fd) commit.

> Besides that, there is a very old DOCS.md file at https://github.com/ionos-cloud/sdk-nodejs/blob/master/DOCS.md which contains more suche cases.

I discussed with the team and we decided to delete those files. I deleted them in [this](https://github.com/ionos-cloud/sdk-nodejs/pull/7/commits/f7770235b9230d61a00cd2981fd17bf150ed7af3) commit.

**Review by mimihalescu — APPROVED** [review]

**Comment by adeatcu-ionos** [comment]
@mimihalescu please review again because I added another [example](https://github.com/ionos-cloud/sdk-nodejs/pull/7/commits/a88d6e5bebf17b17eff894ee637eb209ded3e41c) in the READMEs as a follow up to [this](https://github.com/ionos-cloud/sdk-nodejs/issues/6#issuecomment-1259456857) comment, thank you!

**Review by mimihalescu — APPROVED** [review]
Looks good to me 👍🏻

**Review by cristiGuranIonos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,10 +32,16 @@ npm i --save @ionos-cloud/sdk-nodejs
...
-```javscript
-const sdk = require('@ionos-cloud/sdk-nodejs')
+```const sdk = require('@ionos-cloud/sdk-nodejs')```
+
+Or, if the import is done from an ES module, use:
 
+```import * as sdk from '@ionos-cloud/sdk-nodejs';```
+
+Usage example:
+```javscript
```
Comment (DanielRuf) [inline README.md:44]: Should probably be `javascript`.

Comment (adeatcu-ionos) [reply README.md:44]: @DanielRuf done in https://github.com/ionos-cloud/sdk-nodejs/pull/9/commits/6ee965d01a63236d59dd737f88a78cd94f0d22a1.

**Comment by DanielRuf** [comment]
Perfect, thank you very much =)

**Pull request merged**